### DISCUSSION TITLE
Custom type + custom cell renderer + custom header cell renderer

### DISCUSSION THREAD

**Discussion by vitolpoc** [infinite-table/infinite-react#10] [Ideas]
Add ability to define custom types, custom cell renderer and custom header cell renderer for default and custom types: “string”, “numeric”, “date”, “currency” etc … without having to specify it for each column definition (i.e. as a default configuration) or as a specific column configuration.

**Comment by roblotter** [comment]
We're almost done implementing column types - need to document it thoroughly and it's ready to ship
